src/completeComment.ts

```diff
@@ -56,35 +56,17 @@ export const convertCommentToTxtNode = (
 };
 
 export const isAppearedBeforeNode = (
-  nodeLocation: TxtNodeLineLocation,
-  commentLocation: TxtNodeLineLocation
+  nodeRange: TextNodeRange,
+  commentRange: TextNodeRange
 ): boolean => {
-  if (nodeLocation.start.line >= commentLocation.end.line) {
-    // \begin{document} % comment
-    // \end{document}
-    if (nodeLocation.start.line === commentLocation.end.line) {
-      return nodeLocation.start.column >= commentLocation.end.column;
-    }
-    return true;
-  }
-  return false;
+  return nodeRange[0] >= commentRange[1]; 
 };
 
 export const isIncludedByNode = (
-  nodeLocation: TxtNodeLineLocation,
-  commentLocation: TxtNodeLineLocation
+  nodeRange: TextNodeRange,
+  commentRange: TextNodeRange
 ): boolean => {
-  if (
-    nodeLocation.start.line <= commentLocation.start.line &&
-    nodeLocation.end.line >= commentLocation.end.line
-  ) {
-    // \begin{document}\end{document} % comment
-    if (nodeLocation.end.line === commentLocation.end.line) {
-      return nodeLocation.end.column > commentLocation.start.column;
-    }
-    return true;
-  }
-  return false;
+  return nodeRange[0] <= commentRange[0] && nodeRange[1] >= commentRange[1]
```

Comment:
Is this change the most essential changes in this request?

Reply:
Yes. The previous implementation may cause a bug if a node comes immediately after other node. Those `column` values overlap, but that was not taken into account, I think.

Reply:
It is possible that simply changing `nodeLocation.end.column > commentLocation.start.column` to `nodeLocation.end.column >= commentLocation.start.column` will solve the problem, but it is simpler to implement with TxtNodeRange than to think about this complex implementation.

Reply:
Thank you for the kind comment. I catch the meaning perfectly. I will merge after adding test cases.